extension/vsc-js/extension.js

```diff
@@ -0,0 +1,150 @@
+// The module 'vscode' contains the VS Code extensibility API
```

Comment:
Was this file auto-generated from the Typescript file? Generally in Typescript projects I check in the .ts files but not the .js files. If we go "full-typescript" is this file still needed?